packages/react/src/app-layout/AppLayoutSidebarDialog.tsx

```diff
@@ -167,6 +168,9 @@ function CloseMenuButton({
 				alignItems="center"
 				as={BaseButton}
 				color="action"
+				css={{
+					':focus': { outlineOffset: `-${packs.outline.outlineWidth}` },
```

Comment:
Do we need to do the whole `:focus`/`:focus-visible` dance like in box style.ts?

```
{
	':focus': packs.outline,
	':focus:not(:focus-visible)': { outline: 'none' },
	':focus-visible': packs.outline,
}
```

Reply:
I don't think so as this is just overriding one of the properties from the `packs.outline` object. I also chose `:focus` in case we ever change `focusRingFor="keyboard"` to `all` and it starts picking up press events as well. Also, if `focusRingFor` is removed, and the dev forgets to remove this addition, nothing will break. 

Visual regression tests would be a simpler catch for stuff like this (and the other indent stuff in this PR) but we're not there yet 😅